src/main/java/de/tum/in/www1/artemis/web/rest/dto/CourseManagementDetailViewDTO.java

```diff
@@ -0,0 +1,190 @@
+        return currentPercentageComplaints;
+    }
+
+    public void setCurrentPercentageComplaints(Double currentPercentageComplaints) {
+        this.currentPercentageComplaints = currentPercentageComplaints;
+    }
+
+    public Long getCurrentAbsoluteComplaints() {
+        return currentAbsoluteComplaints;
+    }
+
+    public void setCurrentAbsoluteComplaints(Long currentAbsoluteComplaints) {
+        this.currentAbsoluteComplaints = currentAbsoluteComplaints;
+    }
+
+    public Long getCurrentMaxComplaints() {
+        return currentMaxComplaints;
+    }
+
+    public void setCurrentMaxComplaints(Long currentMaxComplaints) {
+        this.currentMaxComplaints = currentMaxComplaints;
+    }
+
+    public Double getCurrentPercentageMoreFeedbacks() {
+        return currentPercentageMoreFeedbacks;
+    }
+
+    public void setCurrentPercentageMoreFeedbacks(Double currentPercentageMoreFeedbacks) {
+        this.currentPercentageMoreFeedbacks = currentPercentageMoreFeedbacks;
+    }
+
+    public Long getCurrentAbsoluteMoreFeedbacks() {
+        return currentAbsoluteMoreFeedbacks;
+    }
+
+    public void setCurrentAbsoluteMoreFeedbacks(Long currentAbsoluteMoreFeedbacks) {
+        this.currentAbsoluteMoreFeedbacks = currentAbsoluteMoreFeedbacks;
+    }
+
+    public Long getCurrentMaxMoreFeedbacks() {
+        return currentMaxMoreFeedbacks;
+    }
+
+    public void setCurrentMaxMoreFeedbacks(Long currentMaxMoreFeedbacks) {
+        this.currentMaxMoreFeedbacks = currentMaxMoreFeedbacks;
+    }
+
+    public Double getCurrentPercentageAverageScore() {
+        return currentPercentageAverageScore;
+    }
+
+    public void setCurrentPercentageAverageScore(Double currentPercentageAverageScore) {
+        this.currentPercentageAverageScore = currentPercentageAverageScore;
+    }
+
+    public Double getCurrentAbsoluteAverageScore() {
+        return currentAbsoluteAverageScore;
+    }
+
+    public void setCurrentAbsoluteAverageScore(Double currentAbsoluteAverageScore) {
+        this.currentAbsoluteAverageScore = currentAbsoluteAverageScore;
+    }
+
+    public Double getCurrentMaxAverageScore() {
+        return currentMaxAverageScore;
+    }
+
+    public void setCurrentMaxAverageScore(Double currentMaxAverageScore) {
+        this.currentMaxAverageScore = currentMaxAverageScore;
+    }
+
+    public Integer[] getActiveStudents() {
+        return activeStudents;
+    }
+
+    public void setActiveStudents(Integer[] activeStudents) {
+        this.activeStudents = activeStudents;
+    }
+
+    public Course getCourse() {
+        return course;
+    }
+
+    public void setCourse(Course course) {
+        this.course = course;
+    }
+
+    public Integer getNumberOfEditorsInCourse() {
+        return numberOfEditorsInCourse;
+    }
+
+    public void setNumberOfEditorsInCourse(Integer numberOfEditorsInCourse) {
+        this.numberOfEditorsInCourse = numberOfEditorsInCourse;
+    }
```

Comment:
I think it was better if this part was moved to line 62, directly after `setNumberOfTeachingAssistantsInCourse`

Reply:
Ok 👍